app/helpers/bulk_upload_helper.rb

```diff
@@ -0,0 +1,12 @@
+module BulkUploadHelper
+  def bulk_upload_title(controller)
```

Comment:
Would it make sense to only pass controller_name here instead of the entire class?